test/unit/proxy_statuses/proxy_status_versions.rb

```diff
@@ -0,0 +1,34 @@
+require 'test_helper'
+
+class ProxyStatusVersions < ActiveSupport::TestCase
```

Comment:
Same as Tftp, this test doesn't run because of the file & name class lack _test, *Test